{"isMarkdown":true,"isTranslated":false,"isActive":true,"flaws":{},"title":"WebAssembly.Exception.prototype.getArg()","mdn_url":"/en-US/docs/WebAssembly/JavaScript_interface/Exception/getArg","locale":"en-US","native":"English (US)","browserCompat":["webassembly.api.Exception.getArg"],"summary":"The getArg() prototype method of the Exception object can be used to get the value of a specified item in the exception's data arguments.","popularity":0,"modified":"2023-11-20T17:47:35.000Z","source":{"folder":"en-us/webassembly/javascript_interface/exception/getarg","github_url":"https://github.com/mdn/content/blob/main/files/en-us/webassembly/javascript_interface/exception/getarg/index.md","last_commit_url":"https://github.com/mdn/content/commit/d23f8c5c52bdfb6151476a2574e72d323d0d30f4","filename":"index.md"},"short_title":"WebAssembly.Exception.prototype.getArg()","parents":[{"uri":"/en-US/docs/WebAssembly","title":"WebAssembly"},{"uri":"/en-US/docs/WebAssembly/JavaScript_interface","title":"WebAssembly"},{"uri":"/en-US/docs/WebAssembly/JavaScript_interface/Exception","title":"WebAssembly.Exception"},{"uri":"/en-US/docs/WebAssembly/JavaScript_interface/Exception/getArg","title":"WebAssembly.Exception.prototype.getArg()"}],"pageTitle":"WebAssembly.Exception.prototype.getArg() - WebAssembly | MDN","noIndexing":false,"hash":"a8191a006c92d1796d40867f245983040dd01a1cc071d0a5213df5b316a5f3f2"}